opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -280,7 +280,21 @@ def __init__(
         self.start_time = None
 
     def __repr__(self):
```

Comment:
Shouldn't this long version be `__str__`? I mean, to keep `__repr__` as it is and add this new as `__str__`.
Maybe https://stackoverflow.com/questions/1436703/difference-between-str-and-repr could clarify things up.

Reply:
Good point.
Addressed in #156.